assets/js/theme/global/cart-preview.js

```diff
@@ -49,4 +51,53 @@ export default function () {
                 .hide();
         });
     });
+
+    // Check if we can use localStorage
+    function storageAvailable(type) {
```

Comment:
Would prefer to move this out to the top level of this file, or even to `assets/js/theme/common` since we will probably want this for other purposes later.